.github/workflows/generate_chapters.yml

```diff
@@ -75,19 +75,20 @@ jobs:
         echo "regenerating chapters"
         npm run generate
     - name: Create Pull Request
-      uses: peter-evans/create-pull-request@v2
+      id: cpr
+      uses: peter-evans/create-pull-request@v3
```

Comment:
With the new version, they have some breaking changes and one of them is how the author of the PR is assigned. I don't think there is anything wrong in the new style, but if you want to be consistent with previous style then you might want to supply the author input manually as suggested in the [release note](https://github.com/peter-evans/create-pull-request/releases/tag/v3.0.0).

Reply:
Yeah I'm happy to use the real author so actually like this change.